new/LIT7252FetahSeray.xml

```diff
@@ -0,0 +1,103 @@
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-20">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:follows" active="LIT7252FetahSeray" passive="LIT1824Mafteh"/>
+                    <relation name="saws:follows" active="LIT7252FetahSeray#Aganent" passive="LIT1824Mafteh#Monday"><desc>Similar to 
+                        the incipit of <ref type="work" corresp="LIT1824Mafteh#Monday"/> as attested in <ref type="mss" corresp="MEAG002"/>.
+                            </desc></relation>
+                    <relation name="saws:follows" active="LIT7252FetahSeray#Aganent" passive="LIT1824Mafteh#Wednesday"><desc>Similar 
+                        to <ref type="work" corresp="LIT1824Mafteh#Wednesday"/> as attested in <ref type="mss" corresp="MEAG002"/>.
+                            </desc></relation>
+                    <relation name="saws:follows" active="LIT7252FetahSeray#Tor" passive="LIT1824Mafteh#Monday"><desc>Similar to the
+                        explicit of <ref type="work" corresp="LIT1824Mafteh#Monday"/> as attested in <ref type="mss" corresp="MEAG002"/>.
+                            </desc></relation>
+                    <relation name="saws:follows" active="LIT7252FetahSeray#Rakon" passive="LIT1824Mafteh#Tuesday"><desc>Similar to 
+                        the incipit of <ref type="work" corresp="LIT1824Mafteh#Tuesday"/> as attested in <ref type="mss" corresp="MEAG002"/>.
+                            </desc></relation>
+                    <relation name="saws:follows" active="LIT7252FetahSeray#Sefanzer" passive="LIT1824Mafteh#Tuesday"><desc>Similar to 
+                        the explicit of <ref type="work" corresp="LIT1824Mafteh#Tuesday"/> as attested in <ref type="mss" corresp="MEAG002"/>.
+                            </desc></relation>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <note>The edition of the incipits is taken from <ref type="mss" corresp="BLorient13227"/> according to the description in 
+                    <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">132</citedRange></bibl>.</note>
+                <div type="edition" subtype="part" xml:id="Aganent">
+                    <note>Similar to the incipit of <ref type="work" corresp="LIT1824Mafteh#Monday"/> and to
+                        <ref type="work" corresp="LIT1824Mafteh#Wednesday"/> as attested in <ref type="mss" corresp="MEAG002"/>.</note>
+                    <ab> በስመ፡ <gap reason="ellipsis"/> ፍታሕ፡ ሥራየ፡ አጋንንት፡ ወባርያ፡ ሥራየ፡ ካህናት፡ ኪን፡ ወካህናት፡ ወዲያቆናት፡</ab>
```

Comment:
the text parts should be marked up as Geez either individually or in    `<div type="edition">`